docs/tutorial.rst

```diff
@@ -1609,26 +1609,26 @@ best used in conjunction with the ``unthreadable`` decorator.  For example:
 Note that ``@()`` is required to pass the python list ``args`` to a subprocess
 command.
 
-Specification Modifier Aliases
-------------------------------
+Decorator Aliases
+-----------------
 
-Using ``SpecAttrModifierAlias`` and callable ``output_format`` you can
+Using ``DecoratorAlias`` and ``SpecAttrDecoratorAlias`` and callable ``output_format`` you can
```

Comment:
This change introduces for the first time the `DecoratorAlias` but doesn't include it in the example. Should the documentation be expanded to also illustrate when and how the base class should be used, and maybe the relationship between the two?